Include wasm files (#50)
NLESC-JCER/cpp2wasm: CONTRIBUTING.md

    make entangle
    ```

> 7. When `src/js/newtonraphsonwasm.js` has changed, make sure the WebAssembly module is also rebuild with (requires [emscripten](README.md#accessing-c-function-from-JavaScript-in-web-browser) to run).

Reviewer: ```suggestion
7. When `src/js/newtonraphsonwasm.js` changes, WebAssembly module also has to be rebuilt. This will require [emscripten](README.md#accessing-c-function-from-JavaScript-in-web-browser). To rebuild the WebAssembly module run:
```
